src/graphql/resolvers.js

```diff
@@ -392,24 +419,34 @@ module.exports = function Resolvers (ssb) {
     },
 
     Thread: {
-      messages: (parent) => new Promise((resolve, reject) => {
-        pull(
-          pull.values(parent.messages),
-          pullParaMap(publicifyMessage, 5),
-          pull.collect((err, res) => err ? reject(err) : resolve(res))
-        )
-      })
-    },
+      replies: async (parent) => {
+        if (!parent.id) return []
 
-    Comment: {
-      author: (parent) => getProfile(parent.author),
-      replies: (parent) => {
-      },
+        let replies = parent.replies
+
+        if (!replies) {
+          const thread = await getThread(parent.id)
+          replies = thread.replies
+        }
 
+        return new Promise((resolve, reject) => {
+          pull(
+            pull.values(replies),
+            pullParaMap(publicifyMessage, 5),
+            pull.collect((err, res) => err ? reject(err) : resolve(res))
+          )
+        })
+      },
+      author: (parent) => getProfile(parent.author),
       votes: (parent) => getVotes(parent.id),
       votesCount: async (parent) => {
         const votes = await getVotes(parent.id)
         return votes?.length
+      },
+      root: (parent) => {
+        if (!parent.root) return null
+
+        return getThread(parent.root)
```

Comment:
:fire: this doesn't set a `threadMaxSize` ... is that a problem? Could this make for a slow query